src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController+DynamicTableViewModel.swift

```diff
@@ -48,101 +48,111 @@ private extension DynamicAction {
 extension AppInformationViewController {
 	static let model: [Category: (text: String, action: DynamicAction)] = [
 		.about: (
-			text: "App_Information_About_Navigation".localized,
-			action: .push(model: aboutModel, withTitle:  "App_Information_About_Navigation".localized)
+			text: AppStrings.AppInformation.aboutNavigation,
+			action: .push(model: aboutModel, withTitle:  AppStrings.AppInformation.aboutNavigation)
 		),
 		.faq: (
-			text: "App_Information_FAQ_Navigation".localized,
+			text: AppStrings.AppInformation.faqNavigation,
 			action: .safari
 		),
 		.terms: (
-			text: "App_Information_Terms_Navigation".localized,
-			action: .push(model: termsModel, withTitle:  "App_Information_Terms_Navigation".localized)
+			text: AppStrings.AppInformation.termsNavigation,
+			action: .push(model: termsModel, withTitle:  AppStrings.AppInformation.termsNavigation)
 		),
 		.privacy: (
-			text: "App_Information_Privacy_Navigation".localized,
-			action: .push(model: privacyModel, withTitle:  "App_Information_Privacy_Navigation".localized)
+			text: AppStrings.AppInformation.privacyNavigation,
+			action: .push(model: privacyModel, withTitle:  AppStrings.AppInformation.privacyNavigation)
 		),
 		.legal: (
-			text: "App_Information_Legal_Navigation".localized,
-			action: .push(model: legalModel, separators: true, withTitle:  "App_Information_Legal_Navigation".localized)
+			text: AppStrings.AppInformation.legalNavigation,
+			action: .push(model: legalModel, separators: true, withTitle:  AppStrings.AppInformation.legalNavigation)
 		),
 		.contact: (
-			text: "App_Information_Contact_Navigation".localized,
-			action: .push(model: contactModel, withTitle:  "App_Information_Contact_Navigation".localized)
+			text: AppStrings.AppInformation.contactNavigation,
+			action: .push(model: contactModel, withTitle:  AppStrings.AppInformation.contactNavigation)
 		),
 		.imprint: (
-			text: "App_Information_Imprint_Navigation".localized,
-			action: .push(model: imprintModel, withTitle:  "App_Information_Imprint_Navigation".localized)
+			text: AppStrings.AppInformation.imprintNavigation,
+			action: .push(model: imprintModel, withTitle:  AppStrings.AppInformation.imprintNavigation)
 		)
 	]
 }
 
 extension AppInformationViewController {
 	private static let aboutModel = DynamicTableViewModel([
 		.section(
-			header: .image(UIImage(named: "Illu_AppInfo_UeberApp"), height: 230),
+			header: .image(UIImage(named: "Illu_AppInfo_UeberApp"),
+						   //accessibilityLabel: AppStrings.AppInformation.aboutImageDescription,
```

Comment:
Why is this commented out?

Reply:
As above, there should be text for this image, but it's not on the JIRA. I've kept this in but commented for when they supply the accessibility text for it.

Reply:
Should be tagged with TODO, I guess, so that we don't forget about it.

Reply:
Added the TODO for this